build.gradle

```diff
@@ -12,6 +12,8 @@ dependencies {
     compile group: 'org.jetbrains', name: 'annotations', version: '18.0.0'
     compile group: 'org.xerial', name: 'sqlite-jdbc', version: '3.27.2.1'
     testCompile group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.4.0'
+    testCompile("org.assertj:assertj-core:3.11.1")
+    testCompileOnly 'junit:junit:4.12'
```

Comment:
I am not sure why junit Jupiter isn't giving you the dependencies. I guess we can merge this, but when I merge my updated one most of this will change anyway